src/Compilers/Core/Portable/NonNullTypesAttribute.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace System.Runtime.CompilerServices
+{
+    public sealed class NonNullTypesAttribute : Attribute
```

Comment:
Does this need a PROTOTYPE because I'm guessing we're not actually adding this publicly once this ships? (or if so, PublicAPI checker needs some help?)

Reply:
Changed to `internal` and added `// PROTOTYPE` comment to remove.

---
In reply to: [207622255](https://github.com/dotnet/roslyn/pull/29050#discussion_r207622255) [](ancestors = 207622255)